lynxius/client.py

```diff
@@ -60,31 +63,9 @@ def evaluate(self, eval: Evaluator) -> str | None:
         Initiates a batched evaluation job. Returns an eval run ID.
         """
 
-        if isinstance(eval, EvaluatorLocal):
-            raise ValueError(
-                "You are trying to run a remote evaluation with a locally executed "
-                "evaluator. Please use a corresponding remote evaluator class."
-            )
-
-        response = self._client.post(eval.get_url(), json=eval.get_request_body())
-
-        if response.status_code == httpx.codes.CREATED:
-            return response.json()["uuid"]
-        else:
-            print("Error:", response.status_code, response.text)
-            return None
-
-    def store(self, eval: EvaluatorLocal) -> str | None:
-        """
-        Stores the locally executed evaluator on the Lynxius platform.
-        Returns an eval run ID.
-        """
-
-        if not isinstance(eval, EvaluatorLocal):
-            raise ValueError(
-                "You are trying to store an evaluator that is not locally executed. "
-                "Please use a corresponding local evaluator class."
-            )
+        # Local evaluation
+        if self.run_local:
```

Comment:
Does the client need `run_local` flag? What about removing it from the client and doing this instead:
``` python
if eval.run_local:
  eval.evaluate_local()
```
Then we can use the same client for both types of evals.

Reply:
Well spotted! I removed `run_local` from all evals and kept it in the client. Keeping this parameter on the client translates into changing only one line in the code for users if they want to switch from evaluations run locally to evaluations run remotely.